app/controllers/concerns/saml_idp_auth_concern.rb

```diff
@@ -43,7 +43,7 @@ def default_name_id_format
   end
 
   def sp_uses_email_nameid_format?
-    Saml::Idp::Constants::ISSUERS_WITH_EMAIL_NAMEID_FORMAT.include?(current_service_provider.issuer)
+    current_service_provider.allow_email_nameid_format
   end
```

Comment:
would it be worth removing this method entirely now that it's just a single property access?

Reply:
Yeah sure, I can definitely do that - good call!